OJP/OJP_Trips.xsd

```diff
@@ -120,7 +120,7 @@
 					</xs:annotation>
 				</xs:element>
 				<xs:sequence>
-					<xs:element name="ItModesToCover" type="IndividualModesEnumeration" minOccurs="0" maxOccurs="unbounded">
+					<xs:element name="ItModesToCover" type="PersonalModesOfOperationEnumeration" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Shouldn' t that now become `IndividualModeEnumeration` (without "s") rather than `PersonalModesOfOperationEnumeration` - which doesn't contain things like car, cycle, or taxi any more?

Reply:
We have the s in all Enums. I would keep it, or is there opposition? @Aurige @skinkie  @trurlurl

Reply:
You can keep the 's', consistency is key.

Reply:
I agree that having the "s" would be the correct solution. But that's already a step further than what I was stumbling across. What I wanted to say: I think `ItModesToCover` should not be of type `PersonalModesOfOperationEnumeration`, but of type `IndividualModeOperation` as defined elsewhere in this PR.